# display the results if true
# bool
display: true

# display timing information
# bool
display_timing: true
